Block.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Block(){
+    return <div className='box--field'></div>

```

Comment:
naming of classes is very important.

Check 2 things:
1. BEM methodology
2. CSS modules

It will help you to figure out what are the best practices for naming classes in projects